tests/models/test_decoder.py

```diff
@@ -3,7 +3,7 @@
 
 from hepattn.models.decoder import MaskFormerDecoder, MaskFormerDecoderLayer
 
-BATCH_SIZE = 2
+BATCH_SIZE = 1
```

Comment:
I'm a bit wary of specialising to batch size 1 everywhere. For now fine to support local ca stuff with a batch size of 1, but in that case can you set that for those specific tests only, so the rest are still using 2?